applications/DeepSpeed-Chat/training/utils/utils.py

```diff
@@ -173,15 +173,18 @@ def get_optimizer_grouped_parameters(
     model,
     weight_decay,
     lora_lr=5e-4,
-    no_decay_name_list=["bias", "LayerNorm.weight"],
```

Comment:
@mosheisland Thanks for the PR. Since DeepSpeed-Chat supports a variety of models, are we sure that we aren't removing model support in any way by not searching for `LayerNorm.weight` anymore?

Reply:
This change is backward compatible.
After the change "LayerNorm.weight" will be part of no weight decay group.
This is since when comparing parameter names, I use <parameter-name>.lower().
Now, since I have "layernorm.weight" in the modified no_decay_name_list, it will also match "LayerNorm.weight".